src/components/FileTreeNode/index.tsx

```diff
@@ -58,7 +58,7 @@ class FileTreeNodeBase<TreeNodeType> extends React.Component<PublicProps> {
             renderChildNodes()
           ) : (
             <ListGroup.Item
-              className={styles.emptyNodeDirectory}
+              className={makeClassName(styles.node, styles.emptyNodeDirectory)}
```

Comment:
Where does `emptyNodeDirectory` come from? I can't find it.

Reply:
It's just a CSS class, mostly used in unit tests so far. It's possible to declare CSS class names without any CSS in `styles.module.scss`. I used to write an empty definition (`.cls {}`) but a review comment asked me to get rid of that if there was no definition, so 🤷‍♂️

Reply:
> It's possible to declare CSS class names without any CSS in `styles.module.scss`.

Huh, I guess that's why TypeScript didn't catch the typo'd class name in that other patch.